lib/names.py

```diff
@@ -25,6 +25,22 @@ def __init__(self, base):
         self.base = base
         self.base_dot = '.' + base
 
+    @classmethod
+    def create_from_lambda_name(cls, name):
+        """
+        Instantiate AWSNames from the name of a lambda function.  Used by
+        lambdas so they can look up names of other resources.
+
+        Args:
+            name (str): Name of lambda function (ex: multiLambda-integration-boss)
+
+        Returns:
+            (AWSNames)
+
+        """
+        domain = name.split('-', 1)[1]
```

Comment:
Probably need a `.replace('-', '.')` to return the domain back to the original form. This works for referencing other lambdas, but not all other resources.

Reply:
👍

Reply:
👍